### PR TITLE
CloudSQL Recipe

### PR DESCRIPTION
**Pull request by Fryyyyy**
Retrieves contents of log sink for current Cloud SQL instance types (MySQL, PostgreSQL and MSSQL) and additionally extracts log line severity.

### CODE REVIEW

File: tests/lib/processors/gcp_logging_timesketch.py [hunk 1]
```diff
@@ -102,7 +96,9 @@ def testGCEFirewallLog(self):
         'message':
             'User heinz-57@ketchup-research.iam.gserviceaccount.com '
             'performed v1.compute.firewalls.insert '
-            'on projects/ketchup-research/global/firewalls/deny-tomchop-access'
+            'on projects/ketchup-research/global/firewalls/deny-tomchop-access',
+        'severity':
+            'NOTICE'
```
Comment (tomchop) [inline tests/lib/processors/gcp_logging_timesketch.py:101]:
```suggestion
        'severity': 'NOTICE'
```

Same in other places

File: data/recipes/gcp_logging_cloudsql_ts.json [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+        "sketch_id": "@sketch_id"
+      }
+    }
+  ],
+  "args": [
+    [
+      "project_name",
+      "Name of GCP project to collect logs from",
+      null
+    ],
```
Comment (tomchop) [inline data/recipes/gcp_logging_cloudsql_ts.json:38]: Could you have these on one line to be consistent with how the other recipes are formatted?

File: data/recipes/gcp_logging_cloudsql_ts.json [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+      "name": "GCPLogsCollector",
+      "args": {
+        "project_name": "@project_name",
+        "filter_expression": "logName:\"projects/@project_name/logs/cloudsql.googleapis.com\" timestamp>\"@start_date\" timestamp<\"@end_date\""
+      }
+    },
+    {
+      "wants": [
+        "GCPLogsCollector"
+      ],
```
Comment (tomchop) [inline data/recipes/gcp_logging_cloudsql_ts.json:17]: Same here: 

```suggestion
      "wants": ["GCPLogsCollector"],
```

File: data/recipes/gcp_logging_cloudsql_ts.json (hunk 2 above)
Comment (Fryyyyy) [reply data/recipes/gcp_logging_cloudsql_ts.json]: Done

**Pull request merged**